scripts/newinstall.sh

```diff
@@ -45,6 +45,8 @@ help_flag=false
 noop_flag=false
 
 # Use system python to bootstrap unless otherwise specified
+# This is used to install and run EUPS and will not necessarily
+# be the python in the path being used to build the stack itself.
 PYTHON="${PYTHON:-/usr/bin/python}"
```

Comment:
The whole thing is historically weird. I'm attempting to fix the current brokenness. I assume it evolved like this as there was more chance of eups working with an old python than the stack working with an old python. It would be easier if we just checked python and installed miniconda and then used miniconda everywhere. We can't do that if we use eups to install miniconda.

I don't think I object to changing to `$PYTHON` for everything.